src/server/pfs/server/api_server.go

```diff
@@ -39,19 +42,39 @@ import (
 // request structures into normal function calls.
 type apiServer struct {
 	pfs.UnimplementedAPIServer
-	env    Env
-	driver *driver
+	env Env
+
+	// etcdClient and prefix write repo and other metadata to etcd
+	etcdClient *etcd.Client
+	prefix     string
+
+	txnEnv      *txnenv.TransactionEnv
+	storage     *storage.Server
+	commitStore commitStore
+
+	cache *fileset.Cache
 }
 
 func newAPIServer(ctx context.Context, env Env) (*apiServer, error) {
```

Comment:
This seems shorter than everything that was here.

https://github.com/pachyderm/pachyderm/pull/10252/files#diff-a40f20848be655d50f1da1f38f4232133ba02eb4e63e01099055003740955d14L47-L82

I don't see tests failing though, so it's probably fine.